docs/en/status-communities/set-up-your-communitys-approval-options.md

```diff
@@ -6,11 +6,36 @@ hide:
   - navigation
 ---
 
-# Set up your community's approval options
+# Manage community join request
 
- :octicons-tools-24: In progress
+As a community owner, you can manually approve a request from people who wants to join your community. With this option enabled, you receive a notification whenever someone requests to join your community. This way, you can decide whether to accept, reject or ignore the request.
 
-!!! note ""
-     We're working on this content.
+## What to expect
+
+- If a community owner rejects a joining request, the person who sent the request will not be notified.
+- The community screen is where you can find out if your request was approved or denied.
```

Comment:
I'm not sure I understand this part - it reads as if we're addressing the user who sent the request, not the community owner who's reviewing it...